ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.14__amd64-nightly-4.14-upgrade-from-eus-4.12.yaml

```diff
@@ -84,18 +84,17 @@ tests:
           memory: 200Mi
     - chain: openshift-upgrade-qe-test-eus
     workflow: cucushift-installer-rehearse-azure-ipi-workers-rhel8
-- as: baremetalds-ipi-ovn-ipv4-fips-f28
-  cron: 22 2 5 * *
+- as: gcp-ipi-basecap-none-additionalcaps-f14
+  cron: 42 14 1,15 * *
   steps:
-    cluster_profile: equinix-ocp-metal-qe
+    cluster_profile: gcp-qe
     env:
-      DEVSCRIPTS_CONFIG: |
-        FIPS_MODE=true
-        IP_STACK=v4
-        NETWORK_TYPE=OVNKubernetes
+      ADDITIONAL_ENABLED_CAPABILITIES: baremetal marketplace openshift-samples CSISnapshot
+        Console Insights Storage
+      BASELINE_CAPABILITY_SET: None
       FILTERS_ADDITIONAL: ~TELCO&
-      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.12 and @baremetal-ipi'
-      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.12 and @baremetal-ipi'
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.12 and @gcp-ipi and not @fips'
```

Comment:
Per https://github.com/openshift/release/pull/55502#discussion_r1717811321, I guess we do not need the setting, right?

Reply:
as @jiajliu said in other comment:
```
@JianLi-RH you can ignore this one, i just drop it in another pr
```
If you only change the one e2e tag, I will remove all from this pr.

Reply:
okay, then ignore my comments, too.

Reply:
> as @jiajliu said in other comment:

The [comment](https://github.com/openshift/release/pull/55502#discussion_r1718019970) is about removing tags in cpou jobs.

```
$ ls|grep cpou
openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.14.yaml
openshift-openshift-tests-private-release-4.17__amd64-nightly-4.17-cpou-upgrade-from-4.15.yaml
openshift-openshift-tests-private-release-4.17__arm64-nightly-4.17-cpou-upgrade-from-4.14.yaml
openshift-openshift-tests-private-release-4.17__arm64-nightly-4.17-cpou-upgrade-from-4.15.yaml
openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-cpou-upgrade-from-4.14.yaml
openshift-openshift-tests-private-release-4.17__multi-nightly-4.17-cpou-upgrade-from-4.15.yaml
```
I've already removed them in cpou jobs, i think jianlin means no need the tags in the eus jobs in this pr

Reply:
ok, I'll remove them